VATSpy.dat

```diff
@@ -6992,7 +6992,7 @@ KVKX|Potomac Airfield|38.748642|-76.955922|VKX|KZDC|0
 KVLA|Vandalia Muni|38.991466|-89.166189|VLA|KZKC|0
 KVLD|Valdosta Reg. GA|30.7825|-83.276722|VLD|KZJX|0
 KVLL|Oakland/Troy|42.542944|-83.177889||KZOB|0
-KVLY|Harlingen Rio Grande Valley Intl TX|26.228506|-97.654394|VLY|KZHU|1
+KVLY|Rio Grande Valley Approach TX|26.228506|-97.654394|VLY|KZHU|1

```

Comment:
This has the same position and name as the following airport:
KHRL|Harlingen Rio Grande Valley Intl TX|26.228506|-97.654394|HRL|KZHU|0

Wouldnt it be better to add another entry as the following instead, and remove the KVLY?:
KHRL|Harlingen Rio Grande Valley Intl TX|26.228506|-97.654394|VLY|KZHU|1

This avoids us having "fake" airports laying around, and avoids this from happening:
![image](https://user-images.githubusercontent.com/35482228/107195091-2ee92c00-69f1-11eb-8537-2c1d13b16123.png)

Naming something "Approach" shouldn't really be needed as "Approach/Departure" is added on the approach ring anyways.

Reply:
I see what you mean about the approach name. VLY, however, is meant for controllers to log in as VLY_APP, which covers HRL, MFE, and BRO. The confusing part is because the position is Valley Approach, while HRL is known as Harlingen Rio Grande Valley International. Ideally, we wanted VLY to exist as a pseudo airport for controllers to log in while keeping KHRL as its own airport.

Reply:
I've changed the commit by removing KVLY as a field and adding a second KHRL with the VLY Identifier for Controllers to login with, and the respective name, this should clear the duplicate field, while retaining the ability to log in as VLY_APP. Does this look good, or is there any recommended change?